nsds_lab_to_nwb/components/tdt/tdt_manager.py

```diff
@@ -30,9 +30,9 @@ def extract(self, device_name, dev_conf, electrode_table_region):
             result = self.tdt_reader.get_data(device_name)
         else:
             logger.info('- stream {} not found. Available stream list: {}'.format(device_name, stream_list))
-            results = None
+            result = None
             # try alternative (device, stream) name pairs
-            alternative_device_names = [('ECoG', 'Wave')]
```

Comment:
Does this mean 'Wave' is actually equivalent to 'Poly', not 'ECoG'?

Reply:
This pair `('ECoG', 'Wave')` means "the device named 'ECoG' may go with a stream called 'Wave' so let's look for that alternative name" — so I'm pretty sure we can revert this back!

Reply:
Ok my bad. Yes we should revert